package.json

```diff
@@ -2469,6 +2475,7 @@
         "azure-arm-containerregistry": "^3.0.0",
         "azure-arm-website": "^5.3.0",
         "azure-storage": "^2.10.3",
+        "dockerfile-ast": "^0.0.24",
```

Comment:
Need to run `npm update` and get the changes to `package-lock.json`. Also need to verify that the language service isn't affected since it uses 0.0.20.

Reply:
language service seems to be working with this change.